sweetalert2.d.ts

```diff
@@ -832,6 +832,13 @@ declare module 'sweetalert2' {
          * @default null
          */
         onClose?: (modalElement: HTMLElement) => void;
+        
+        /**
+         * Set to false to disable body padding adjustment when scrollbar is present. 
+         *
+         * @default false
```

Comment:
copy-paste typo I guess, should be `true`

Reply:
![image](https://user-images.githubusercontent.com/17089396/52916125-b3f24c80-32d3-11e9-8a78-c4eb7d0a7a1c.png)

done in https://github.com/sweetalert2/sweetalert2/pull/1414/commits/11a7e68e70f924a186bc04139a8352b16a96e3e4